draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -552,7 +552,7 @@ set-cookie        = set-cookie-string
 set-cookie-string = BWS cookie-pair *( BWS ";" OWS cookie-av )
 cookie-pair       = cookie-name BWS "=" BWS cookie-value
 cookie-name       = token
-cookie-value      = *cookie-octet / ( DQUOTE *cookie-octet DQUOTE )
+cookie-value      = ( DQUOTE *cookie-octet DQUOTE ) / *cookie-octet
```

Comment:
This change is pointless.  It is exactly the same as the original.  (Yes, I've read the erratum: people writing parsers that depend on order can rewrite the ABNF for themselves.)

Reply:
I do suspect that anyone writing a parser in the last ~15 years has probably either not had this problem, or solved it. Changing the spec doesn't seem terribly helpful at this point, and I agree with @martinthomson that this is the same from a grammatical standpoint so the churn it introduces seems confusing without reading the errata. 🤷  Optimizing for parsers' potential ordering issues doesn't seem necessary.

Reply:
Works for me, I'll revert the change.